### ISSUE TITLE
[SUGGESTION] Provide way to get needed workloads

### ISSUE THREAD

**Issue by webreidi** [microsoft/vscode-dotnettools#1367]
### Describe the feature you'd like

When a user opens a repo and my system does not have the required workloads, I get a message that tells me that it failed to restore the solution and gives me the command I need to run to get the workloads. It would be great if it would provide me with a prompt allowing me to install the workloads without having to jump to the terminal to run a command.

-- Feedback from internal user

**Comment by nagilson** [comment]
I think our plan was to auto-install the workloads... is a prompt better? 

cc @baronfel @marcpopMSFT

**Comment by webreidi** [comment]
@adrianhall was the original requestor. I'd love his thoughts.

Also, would love the MAUI folks input on whether they think auto installing would be acceptable.

**Comment by marcpopMSFT** [comment]
We should consult with the Maui team on this. My original thought on this is that the Maui extension would ask us to install the workloads is requires and we would do that. The reason to do it this way is we don't know the list of workloads the customer will need in Maui devkit scenarios (we could hard code it but it could change). Whether the Maui extension asked the customer first or just did it on activation, I was going to leave to them.

**Comment by nagilson** [comment]
Yes, I imagine we will be working with the Aspire team as well when we develop this feature 😉
